### PR TITLE
Can install mono on CentOS6

### PR DESCRIPTION
**Pull request by gturri**
- Package name comes from http://www.rpmfind.net/linux/rpm2html/search.php?query=mono-complete
- Repo url and key come from http://www.mono-project.com/docs/getting-started/install/linux/#centos-fedora-and-derivatives

### CODE REVIEW

File: recipes/install_packages.rb [hunk 1]
```diff
@@ -23,4 +23,12 @@
   package "mono-stable"
 when "debian", "ubuntu"
   package "mono-complete"
+when "rhel"
+  yum_repository 'mono' do
+    description "Mono repository"
+    baseurl node['mono']['yum_repository']
+    gpgkey node['mono']['yum_repository_key']
+    action :create
```
Comment (kamaradclimber) [inline recipes/install_packages.rb]: `:create` is the default action, you can skip it

File: attributes/default.rb [hunk 2]
```diff
@@ -6,4 +6,7 @@
 when "debian","ubuntu"
   default['mono']['prefix']	=	"/usr/local"
```
Comment (jmauro) [inline attributes/default.rb:6]: Does thoses settings also possible for rhel?

Regards,
JM

Comment (kamaradclimber) [inline attributes/default.rb:6]: `prefix` is used only when building from source

**Comment by jmauro** [comment]
Ok to merge for the package usage.

**Pull request merged**